reana_client/cli/workflow.py

```diff
@@ -1069,22 +1050,22 @@ def workflow_run(
     # set context parameters for subcommand
     ctx.invoked_by_subcommand = True
     ctx.workflow_name = ""
-    click.secho("[INFO] Creating a workflow...", bold=True)
+    display_message("[INFO] Creating a workflow...")
     ctx.invoke(
         workflow_create,
         file=file,
         name=name,
         skip_validation=skip_validation,
         access_token=access_token,
     )
-    click.secho("[INFO] Uploading files...", bold=True)
+    display_message("[INFO] Uploading files...")
```

Comment:
The same as above

Reply:
Done in https://github.com/reanahub/reana-client/pull/553/commits/b04b1d606068ae6ca9fb898509f26fcabf43b1cd